scopes/harmony/config/workspace-config.ts

```diff
@@ -137,6 +140,13 @@ export class WorkspaceConfig implements HostConfig {
     return isChanged;
   }
 
+  getExistingKeyIgnoreVersion(id: ComponentID): string | undefined {
+    const idStr = id.toStringWithoutVersion();
+    if (this.raw[idStr]) return idStr;
+    const keys = Object.keys(this.raw);
+    return keys.find((key) => key.startsWith(`${idStr}@`));
```

Comment:
If the component extension is part of the ws you might have a key without a version at all.
Then the match with @ won't find it.
So maybe add || exactMatch (without the @)

Reply:
@GiladShoham , this is handled in lines 144, 145 
```
const idStr = id.toStringWithoutVersion();
if (this.raw[idStr]) return idStr;
```